e2e/cypress/support/commands.ts

```diff
@@ -45,25 +45,28 @@ Cypress.Commands.add('logout', () => {
 // if there are no resulting changes to apply, it will simply click out of the filter.
```

Comment:
I am guessing that you found it more reliable to alter the strategy here to always clear the filter and always apply the changes. Any idea why that is the case? It is better to let controls behave "naturally" (the way a user would use them) rather than have to force them, but I understand it may be necessary sometimes.
However, if that is the way we need to go, please update the comment on this line to match.

Reply:
The way this command was originally written proved flaky in cases when the same projects were checked over and over - it would just hang after checking the boxes.

Since this helper will be used in a variety of tests with different preconditions that will be tricky to predict, I'd prefer to make sure of the `force:true` shortcut for reliability. I'd rather expand the `global_filter.spec.ts` to add tests covering specific behavior expectations where we have more control over what state the project dropdown will begin with.